src/plugins/intel_cpu/src/nodes/gather.cpp

```diff
@@ -130,10 +132,18 @@ void Gather::initSupportedPrimitiveDescriptors() {
 
     // Implementation desc type will be redefined in the fn prepareParams if a kernel will be created.
     ov::element::Type dataPrecision = getOriginalInputPrecisionAtPort(GATHER_DATA);
+
+    ov::element::Type outputPrecision = getOriginalOutputPrecisionAtPort(0);
```

Comment:
this initial value `getOriginalOutputPrecisionAtPort(0)` is not used, suggest to initialize `outputPrecision` with`dataPrecision` and in line 146 we can directly use `outputPrecision` for output port0

Reply:
Done